pkg/validation/validation.go

```diff
@@ -0,0 +1,61 @@
+package validation
+
+import (
+	"fmt"
+	"regexp"
+	"strings"
+
+	"github.com/pkg/errors"
+)
+
+// The consts and vars beginning with dns* were taken from: https://github.com/kubernetes/apimachinery/blob/fc49b38c19f02a58ebc476347e622142f19820b9/pkg/util/validation/validation.go
+const dns1123LabelFmt string = "[a-z0-9]([-a-z0-9]*[a-z0-9])?"
```

Comment:
I would use `const ( ... )` and `var ( ... )` for multiple const and vars.

Reply:
Done